crates/proof-of-sql/src/base/commitment/commitment_evaluation_proof_test.rs

```diff
@@ -140,141 +143,141 @@ pub fn test_random_commitment_evaluation_proof<CP: CommitmentEvaluationProof>(
 #[test]
 #[cfg(feature = "blitzar")]
 fn test_simple_ipa() {
-    test_simple_commitment_evaluation_proof::<InnerProductProof>(&(), &());
+    test_simple_commitment_evaluation_proof::<NaiveEvaluationProof>(&(), &());
```

Comment:
For all of these tests, I would like to see them _copied_ to the `inner_product` module.
This is the most important test set to ensure is run on every commitment scheme, because it does a pretty thorough end-to-end tests.

It's fine to _also_ keep them here, and I actually prefer that.

Since `NaiveEvaluationProof` doesn't require `blitzar`, you can also remove the `#[cfg(feature = "blitzar")]` annotation here.

Reply:
Okay, copied the tests. Using `InnerProductProof`

But what I see, these tests are exactly the same as the tests in `naive_evaluation_proof.rs`, So I removed them from `commitment_evaluation_proof_test.rs`